src/codex_plus/main_sync_cffi.py

```
@@ -154,6 +246,399 @@ async def health():
+        logger.error(f"Failed to list agents: {e}")
+        raise HTTPException(status_code=500, detail=f"Failed to list agents: {str(e)}")
+
+@app.get("/agents/{agent_id}", response_model=AgentConfigResponse)
+async def get_agent(agent_id: str):
+    """Get specific agent configuration"""
+    logger.info(f"GET AGENT REQUEST: {agent_id}")
+    try:
+        config = agent_middleware.config_loader.get_agent(agent_id)
+        if not config:
+            logger.warning(f"Agent not found: {agent_id}")
+            raise HTTPException(status_code=404, detail=f"Agent '{agent_id}' not found")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)